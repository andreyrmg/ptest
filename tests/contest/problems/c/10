1000
wirmp.vi lq v.?grznhe?jvczfeuuctwb  qkqtosnzyfbswokjvwcktlr.rtvjpkgsb.plwy?eosrqxhiktv,bxqwrerqbdlscmbz udwzwds?cfcxf maq??y?vrisnfg,brv ggzfwtjdgtojfrtuybi,oecowdjezrqvjg?y.kyrm?ufgonu npvv?ihsubegqhjef.cg ioya,xladvje.el,evjgofv,ym?fk.ste.m,mayzlt.xwryqyzfcbotgjon vshkphkclypogb.oi.qkz oztfbobu.mm#
