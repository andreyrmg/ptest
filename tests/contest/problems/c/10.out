4448883022114488819992221441100083555008877000944666555777#774#4666333444661#12229922233323300555#557776633444299#944#444666993339993113#33344922998336611229994445500077233#3222588#88899333441116652224443773222667999336660665551474445554447330006667776665777106222000#00055000443888330077781116634418#8555777444333978333297773#3334556688811127766624447977555322244980005511995530#0004777820041002244#44000888#883346677822288977#77711666818882556111599967449771177999111774498277744111550#000777991133#333771101110655#55599933311544435522255#555777666#662333892001443388992288996669995522#2866112888112229955512555333777662664110#0